src/inference/gibbs_conditional.jl

```diff
@@ -0,0 +1,135 @@
+"""
+    GibbsConditional(sym, conditional)
+
+A "pseudo-sampler" to manually provide analytical Gibbs conditionals to `Gibbs`.
+`GibbsConditional(:x, cond)` will sample the variable `x` according to the conditional `cond`, which
+must therefore be a function from a `NamedTuple` of the conditioned variables to a `Distribution`.
+
+
+The `NamedTuple` that is passed in contains all random variables from the model in an unspecified
+order, taken from the [`VarInfo`](@ref) object over which the model is run.  Scalars and vectors are
```

Comment:
```suggestion
order, taken from the [`VarInfo`](@ref) object over which the model is run. Scalars and vectors are
```

Reply:
Just to mention that: the convention of separating sencences with double spaces didn't come naturally to me (it's an American idea, I have heard) -- but it makes navigation in Emacs easier, so I kind of do it automatically now :D

Reply:
Lol no, so many Americans hate this too. There was kind of a big culture war about this recently. My preference is for single space between sentences, and I'd rather not choose spacing rules on the margin of "what makes emacs people happy".